cli.py

```diff
@@ -0,0 +1,146 @@
+# NEW_FILE_PATH: /venvs/judges/source/judges/cli.py
+
+import typer
+import json
+import os
+from typing import Dict, List
+from judges import choices_judges, get_judge_by_name
+
+app = typer.Typer()
+
+
+def parse_json_dict(json_dict: str) -> List[Dict[str, str]]:
+    """
+    Parse a JSON dictionary or path to a JSON file into a list of dictionaries.
+    Each dictionary must have 'input', 'output', and 'expected' keys.
+
+    Args:
+        json_dict: Either a JSON string or a path to a JSON file
+
+    Returns:
+        List of dictionaries with 'input', 'output', and 'expected' keys
+
+    Raises:
+        ValueError: If the JSON is invalid or missing required keys
+    """
+    # Try to parse as JSON string first
+    try:
+        data = json.loads(json_dict)
+    except json.JSONDecodeError:
+        # If not a valid JSON string, try to read as file
+        if not os.path.exists(json_dict):
+            raise ValueError(f"Invalid JSON string and file not found: {json_dict}")
+        try:
+            with open(json_dict, "r") as f:
+                data = json.load(f)
+        except json.JSONDecodeError:
+            raise ValueError(f"Invalid JSON in file: {json_dict}")
+
+    # Convert single dictionary to list
+    if isinstance(data, dict):
+        data = [data]
+
+    # Validate format
+    required_keys = {"input", "output", "expected"}
+    for i, entry in enumerate(data):
+        if not isinstance(entry, dict):
+            raise ValueError(f"Entry {i} is not a dictionary")
+
+        # Check for missing keys
+        missing_keys = required_keys - set(entry.keys())
+        if missing_keys:
+            raise ValueError(f"Entry {i} is missing required keys: {missing_keys}")
+
+        # Check for empty strings
+        for key in required_keys:
+            if entry[key] == "":
+                print(f"Warning: Empty string found for key '{key}' in entry {i}")
+
+    return data
+
+
+@app.command()
+def main(judge: choices_judges, model_name: str, json_dict: str, out: str = None):
```

Comment:
```suggestion
def main(judge: choices_judges, model_name: str, json_dict: str, output: str = None):
```

It'd also be great to have a commonly used shortcut of `-o` mapping to `output`

Reply:
Integrated shortcuts for model, output and input.